### PR TITLE
add team 5 html

### PR DESCRIPTION
**Pull request by BThero**
@lintin9095 @ysm0706glee @trackmystories @BThero @Scientist-Ansh 
Please review if everything is OK

### CODE REVIEW

File: teams/5/blog-posts.html [hunk 1]
```diff
@@ -0,0 +1,503 @@
...
+        <p style="position: absolute; left: 0; top: 0;">&lt;section&gt;</p>
+
+        <div class="container">
+          <p style="position: absolute; left: 0; top: 0;">&lt;div&gt;</p>
+
+          <h2 class="text-big">&lt;h2&gt;</h2>
+          <p class="text">&lt;p&gt;</p>
+
+          <div class="buttons">
+            <p style="position: absolute; left: 0; top: 0;">&lt;div&gt;</p>
```
Comment (ysm0706glee) [inline teams/5/blog-posts.html:171]: Maybe <button></button>

File: teams/5/blog-posts.html [hunk 2]
```diff
@@ -0,0 +1,503 @@
...
+            <p class="button button-1">&lt;button&gt;</p>
```
Comment (ysm0706glee) [inline teams/5/blog-posts.html:172]: Maybe <button></button>

**Comment by Scientist-Ansh** [comment]
Lastly, in settings.html you can make do with just 2 form tags I think. One for personal info and other for profile.

Awesome work guys🥳🥳🥳🥳

File: teams/5/dashboards.html [hunk 3]
```diff
@@ -0,0 +1,459 @@
...
+
+      <a href="#">&lt;a&gt;<img src="#" alt="&lt;img&gt;"></a>
+    </nav>
+
+    <!-- Main content -->
+    <main>
+      <p style="position: absolute; left: 0; top: 0;">&lt;main&gt;</p>
+
+      <!-- Heading -->
+      <section id="heading">
```
Comment (ysm0706glee) [inline teams/5/dashboards.html:82]: header instead of section?

File: teams/5/settings.html [hunk 4]
```diff
@@ -0,0 +1,290 @@
...
+
+
+
+
+  </section>
+
+
+  </main>
+</body>
+</html>
```
Comment (ysm0706glee) [inline teams/5/settings.html:290]: I think it's good but maybe it's easier to read if indentations are the same 🤔

**Review by ysm0706glee — COMMENTED** [review]
I added some comments so please check them put and let me know what you think.

File: teams/5/dashboards.html (hunk 3 above)
Comment (BThero) [reply teams/5/dashboards.html:82]: Grace said it is fine

**Review by lintin9095 — DISMISSED** [review]
ok

File: teams/5/blog-posts.html (hunk 2 above)
Comment (BThero) [reply teams/5/blog-posts.html:172]: thanks fixed

**Comment by BThero** [comment]
@trackmystories hey Ali, your page has a lot of html syntax errors. Format it using prettier please.

File: teams/5/settings.html [hunk 5]
```diff
@@ -0,0 +1,290 @@
...
+          <li><div class="row"><i>i</i><a href="#">ul li a</a> <span>span</span>  </div></li>
+          <li><div class="row"><i>i</i><a href="#">ul li a</a> <span>span</span>  </div></li>
+          <li><div class="row"><i>i</i><a href="#">ul li a</a> <span>span</span>  </div></li>
+          <li><div class="row"><i>i</i><a href="#">ul li a</a> <span>span</span>  </div></li>
+   
+        </ul>  
+       
+         <ul>       
+          <p>ul</p>  
+          <li><div class="spaceTopTwo"><i>i</i><a href="#">ul li a</a></div></li>
```
Comment (lintin9095) [inline teams/5/settings.html:52]: Is this the Name + Email part in the bottom of the second nav? If yes, I don't think it should be a list but just span

File: teams/5/settings.html [hunk 6]
```diff
@@ -0,0 +1,290 @@
...
+          <li><div><i>i</i><a href="#">ul li a</a></div></li>
+         <li>
+           <div class="row"></div>
+             <div  class="spaceTopTwo">
+             <h2>h2</h2>
```
Comment (lintin9095) [inline teams/5/settings.html:57]: what is this h2 and p?

File: teams/5/settings.html [hunk 7]
```diff
@@ -0,0 +1,290 @@
...
+          <li><div class="row"><i>i</i><a href="#">ul li a</a> <span>span</span>  </div></li>
+   
+        </ul>  
+       
+         <ul>       
+          <p>ul</p>  
+          <li><div class="spaceTopTwo"><i>i</i><a href="#">ul li a</a></div></li>
+          <li><div><i>i</i><a href="#">ul li a</a></div></li>
+         <li>
+           <div class="row"></div>
```
Comment (lintin9095) [inline teams/5/settings.html:55]: why do we have an empty div here?

File: teams/5/settings.html [hunk 8]
```diff
@@ -0,0 +1,290 @@
...
+        </ul>  
+      </div>
+    </nav> 
+
+ <main>
+    <div class="mainHead">
+     <p>main div</p>
+    </div>
+
+  <header>
```
Comment (lintin9095) [inline teams/5/settings.html:72]: Should we wrap this whole part to <section>?

File: teams/5/settings.html [hunk 9]
```diff
@@ -0,0 +1,290 @@
...
+        <div>
+        <label>form label</label>
+        <input type="text">
+        <label>label</label>
+        <input type="text">
+        </div>
+
+        <div>
+        <label>label</label>
+        <input type="text">
```
Comment (lintin9095) [inline teams/5/settings.html:120]: we are missing the tag for email icon here

File: teams/5/settings.html [hunk 10]
```diff
@@ -0,0 +1,290 @@
...
+        </div>
+
+      </form>
+
+
+      <div>
+          <img src="">
```
Comment (lintin9095) [inline teams/5/settings.html:127]: is this the icon for "Upload". It should be <i>

File: teams/5/settings.html [hunk 11]
```diff
@@ -0,0 +1,290 @@
...
+          <div>
+            <p>img div p</p>
+            <p>p</p>
```
Comment (lintin9095) [inline teams/5/settings.html:130]: missing a link/button for "Click to upload"

File: teams/5/settings.html [hunk 12]
```diff
@@ -0,0 +1,290 @@
...
+          </div>
```
Comment (lintin9095) [inline teams/5/settings.html:131]: this is actually 2 separated lines. Should have another <span> here

File: teams/5/settings.html [hunk 13]
```diff
@@ -0,0 +1,290 @@
...
+        <label>label</label>
+        <input type="text">
+        </div>
+
+        <div>
+        <label>label</label>
+        <input type="text">
+        </div>
+
+      </form>
```
Comment (lintin9095) [inline teams/5/settings.html:123]: is it correct to close the </form> here?

File: teams/5/settings.html [hunk 14]
```diff
@@ -0,0 +1,290 @@
...
+      <button>Button</button>
+    </div>
+
+  
+        </div>
+
+
+
+
+       <div class="upload">
```
Comment (lintin9095) [inline teams/5/settings.html:143]: what is this div for????

File: teams/5/settings.html [hunk 15]
```diff
@@ -0,0 +1,290 @@
...
+
+     <img src="ms.jpg" alt="img" width="50px" height="50px">
+       <form action="/action_page.php">
+       <input type="file" id="myFile" name="filename">
+       <input type="submit">
+       </form>
+       </div> 
+
+
+    </div>
```
Comment (lintin9095) [inline teams/5/settings.html:153]: no opening tag for this enclosing tag

File: teams/5/settings.html [hunk 16]
```diff
@@ -0,0 +1,290 @@
...
+
+    
+  </div>
+  </section>
+
+  <div class="margin" ></div>
```
Comment (lintin9095) [inline teams/5/settings.html:159]: this can be an <hr> instead of <div>

File: teams/5/settings.html [hunk 17]
```diff
@@ -0,0 +1,290 @@
...
+
+
+  <section>
+
+  <div class="row">
```
Comment (lintin9095) [inline teams/5/settings.html:164]: this whole switch part belong to the "right" part.

File: teams/5/settings.html [hunk 18]
```diff
@@ -0,0 +1,290 @@
...
+</label>
+
+<div>
+  <h2>h2</h2>
+  <p>p</p>
+</div>
+</div>
+
+
+  <div class="row">
```
Comment (lintin9095) [inline teams/5/settings.html:178]: what is this?????????????

File: teams/5/settings.html [hunk 19]
```diff
@@ -0,0 +1,290 @@
...
+      <div class="row">
+      <span>span</span>  
+      <input type="text">
+      </div>
+      <label>label</label>
+      <div class="row">
+      <span>span</span>  
+      <input type="text">
+      </div>
+    </form>
```
Comment (lintin9095) [inline teams/5/settings.html:213]: is it correct to close the </form> here?

File: teams/5/settings.html [hunk 20]
```diff
@@ -0,0 +1,290 @@
...
+
+    <p>p</p>
```
Comment (lintin9095) [inline teams/5/settings.html:215]: this is unnecessary. Please delete

File: teams/5/settings.html [hunk 21]
```diff
@@ -0,0 +1,290 @@
...
+
+</div>
+
+
+
+      <textarea id="w3review" name="w3review" rows="4" cols="50">
+       web development technologies. web development technologies. web development technologies. web development technologies. web development technologies.
+       </textarea>
+
+       <p>p</p>
```
Comment (lintin9095) [inline teams/5/settings.html:225]: This should be a span

File: teams/5/settings.html [hunk 22]
```diff
@@ -0,0 +1,290 @@
...
+
+    <div class="row_2">
+
+    <form>
+      <label>label</label>
+      <input type="text">
```
Comment (lintin9095) [inline teams/5/settings.html:231]: This is for "Normal Text" dropdown, right? Then it should not be type="text"

File: teams/5/settings.html [hunk 23]
```diff
@@ -0,0 +1,290 @@
...
+
+      <textarea id="w3review" name="w3review" rows="4" cols="50">
+       web development technologies. web development technologies. web development technologies. web development technologies. web development technologies.
+       </textarea>
+
+       <p>p</p>
+
+    <div class="row_2">
+
+    <form>
```
Comment (lintin9095) [inline teams/5/settings.html:229]: Everything should be wrapped in 1 form, so this is unnecessary

File: teams/5/settings.html [hunk 24]
```diff
@@ -0,0 +1,290 @@
...
+      <label>label</label>
+      <input type="text">
+    </form>
+
+
+
+    
+
+    <div class="row_2"><i>i</i> <i>i</i> <i>i</i> <i>i</i> <i>i</i></div>
```
Comment (lintin9095) [inline teams/5/settings.html:238]: all of these "i" should be buttons

File: teams/5/settings.html [hunk 25]
```diff
@@ -0,0 +1,290 @@
...
+
+    </div>
+    
+
+
+
+
+    <div> 
+      <select name="normal text" >
```
Comment (lintin9095) [inline teams/5/settings.html:247]: The flag icon/img is missing here

File: teams/5/settings.html [hunk 26]
```diff
@@ -0,0 +1,290 @@
...
+        <option value=""></option>
+        <option value=""></option>
+      </select>
+    </div>
+
+    <div> 
+      <label>label</label>
+
+      <select>
+        <option>option</option>
```
Comment (lintin9095) [inline teams/5/settings.html:259]: "clock" icon missing

File: teams/5/settings.html (hunk 6 above)
Comment (trackmystories) [reply teams/5/settings.html:57]: I don't understand your question, kindly clarify.

File: teams/5/settings.html (hunk 8 above)
Comment (trackmystories) [reply teams/5/settings.html:72]: dont understand your question please clarify, wrap in what?

File: teams/5/settings.html (hunk 12 above)
Comment (trackmystories) [reply teams/5/settings.html:131]: please clarify

File: teams/5/settings.html [hunk 27]
```diff
@@ -0,0 +1,290 @@
...
+        </div>
+
+        <div>
+        <label>label</label>
+        <input type="text">
+        </div>
+
+      </form>
+
+
```
[the remark this answers is not in the document]
Comment (trackmystories) [reply teams/5/settings.html:125]: please share screenshot, im not sure what you are referring to.

File: teams/5/settings.html (hunk 13 above)
Comment (trackmystories) [reply teams/5/settings.html:123]: what is correct?

File: teams/5/settings.html (hunk 14 above)
Comment (trackmystories) [reply teams/5/settings.html:143]: for styling purposes

Comment (BThero) [reply teams/5/settings.html:143]: You can actually style things without those extra <div>s. Try keeping your code as clean as possible.

File: teams/5/settings.html (hunk 6 above)
Comment (lintin9095) [reply teams/5/settings.html:57]: from my understanding, in this part you create the bottom part for sub nav, which includes: 
1 name
1 email
1 icon for log out

This should be translated into: 
2 span + 1 i

But from your code, there are: 
2 i => not sure what is this
then 1 h2 => should be span
then 1 p => should be span
then 1 i => this is correct

**Comment by trackmystories** [comment]
please the changes made, I have also run them through the W3 validation.

**Review by ysm0706glee — DISMISSED** [review]

**Review by lintin9095 — APPROVED** [review]

**Review by trackmystories — APPROVED** [review]
Every thing is good guys, excellent stuff.

**Review by ysm0706glee — APPROVED** [review]

**Pull request merged**